explainer.md

```diff
@@ -206,7 +206,7 @@ Ensuring context compatibility with a `VRDevice` through either method may have
 
 WebVR provides information about the current frame to be rendered via the `VRPresentationFrame` object which developers must examine each frame. The `VRDevicePose` contains the information about all views which must be rendered and targets into which this rendering must be done.
 
-`VRWebGLLayer` objects are not updated automatically. To present new frames, developers must use `VRSession`'s `requestFrame()` method. When the callback function is run, it passes fresh rendering data that must be used to draw into the `VRWebGLLayer`'s `framebuffer` during the callback. This framebuffer is created by the UA and behaves similarly to a canvas's default framebuffer. Using framebufferTexture2D, framebufferRenderbuffer, getFramebufferAttachmentParameter, and getRenderbufferParameter will all generate an INVALID_OPERATION error. Additionally, attempting to render to this framebuffer outside of the `requestFrame()` callback will generate an INVALID_OPERATION error. 
+`VRWebGLLayer` objects are not updated automatically. To present new frames, developers must use `VRSession`'s `requestAnimationFrame()` method. When the callback function is run, it is passed both a timestamp (identical to the one given by `window.requestAnimationFrame()`'s callback) and a `VRPresentationFrame` containing fresh rendering data that must be used to draw into the `VRWebGLLayer`'s `framebuffer` during the callback. This framebuffer is created by the UA and behaves similarly to a canvas's default framebuffer. Using `framebufferTexture2D`, `framebufferRenderbuffer`, `getFramebufferAttachmentParameter`, and `getRenderbufferParameter` will all generate an INVALID_OPERATION error. Additionally, attempting to render to this framebuffer outside of the `requestAnimationFrame()` callback will generate an INVALID_OPERATION error. 
```

Comment:
> identical to the one given by `window.requestAnimationFrame()`'s callback

Not to be pedantic, but this could mean a lot of things.  I'm a bit worried that people will interpret it as being the same number as the last window.RAF or something.  IIRC we'd chatted about returning 0 for right now so folks wouldn't be enticed to use the data?

Reply:
Better?